mesheryctl/internal/cli/root/system/dashboard.go

```diff
@@ -22,15 +22,29 @@ import (
 var (
 	// runPortForward is used for port-forwarding Meshery UI via `system dashboard`
 	runPortForward bool
+)
 
-	// defaultHost is the default host used for port-forwarding via `system dashboard`
-	defaultHost = "localhost"
-
-	// defaultPort is for port-forwarding via `system dashboard`
-	defaultPort = 9081
+// dashboardOptions holds values for command line flags that apply to the dashboard
+// command.
+type dashboardOptions struct {
+	host    string
+	port    int
+	podPort int
+}
 
-	podPort = 8080
-)
+// newDashboardOptions initializes dashboard options with default
+// values for host, port, and which dashboard to show. Also, set
+// max wait time duration for 300 seconds for the dashboard to
+// become available
+//
+// These options may be overridden on the CLI at run-time
+func newDashboardOptions() *dashboardOptions {
+	return &dashboardOptions{
+		host:    "localhost",
```

Comment:
@piyushsingariya what I meant is that these values are defined elsewhere already and should not be repeated here. Please reference the centralized values.

Reply:
I'm about to make more changes, this is not ready for review yet.

Reply:
I know what you meant, I didn't finished the work yet!